### ISSUE TITLE
AC-7 Unsuccessful Logon attempts

### ISSUE THREAD

**Issue by Abi-Nada** [cds-snc/platform-forms-client#852]
### Security Story

**Story context**
Risk flagged from Security Assessment Report: Lock account used to logon to application after x # of failed attempts where x is defined by CDS

**Acceptance criteria**
- [ ] Identify which accounts to be locked
- [ ] Define number of failed attempts
- [ ] Connect with Notify to discuss how they solve this problem
- [ ] Write a development tasking card
- [ ] Identify how clients can reactivate their account
- [ ] risk register, SA&A, and SRTM are updated to reflect new risks and their treatments. Treatments are implemented or a new card is created.

**Comment by craigzour** [comment]
@Abi-Nada @srtalbot @sarahhobson I know you wanted to have a discussion about this ticket once the implementation is done. Should we wait for the PR (https://github.com/cds-snc/platform-forms-client/pull/915) to be approved and merged first?

**Comment by srtalbot** [comment]
Thanks for flagging, @craigzour. There are some updates to the Security Assessment and Authorization and Security requirements traceability matrix. Basically they're documentation that we're meeting security controls. 

Could you please go to the [access control and authentication policy in the AC-7 heading](https://docs.google.com/document/d/1_TVx2mPb1tmsj0D-0HDSMtavzLX21cltJJzk43_duTA/edit#heading=h.mw7lcweqenu80 and write a short explanation of what the number of failed attempts are and how someone would reset. Also provide a link to the codebase where the code is, please.

**Comment by craigzour** [comment]
@srtalbot The document has been updated https://docs.google.com/document/d/1_TVx2mPb1tmsj0D-0HDSMtavzLX21cltJJzk43_duTA/edit#heading=h.mw7lcweqenu8

I will let you close the ticket if there is no need to add more information.
